working/augmentation-libraries/feature-specification.md

```diff
@@ -573,8 +571,8 @@ It is a compile-time error if:
     a variable with an initializer.
 
 *   A final variable is augmented with a setter. (Instead, the augmentation
-    library can declare a *non-augmenting* setter that goes alongside the
-    implicit getter defined by the final variable.)
+    can declare a *non-augmenting* setter that goes alongside the implicit
+    getter defined by the final variable.)
```

Comment:
(This is the general rule of "you can't augment what doesn't exist", and a final variable doesn't introduce a setter, so an `augment` setter declaration cannot augment what isn't there. Right?)

Reply:
Yes, I don't think it hurts to explicitly call it out though? But that is the reasoning for sure.